lib/logstash/outputs/elasticsearch/elasticsearch-template.json

```diff
@@ -5,88 +5,37 @@
   },
   "mappings" : {
     "_default_" : {
-      "_all" : {"enabled" : true, "omit_norms" : true},
```

Comment:
Should we keep disabling norms? With the new syntax, this would be `"_all" : {"norms" : false}`